contracts/upgradeable_contracts/native_to_erc20/HomeBridgeNativeToErc.sol

```diff
@@ -42,7 +48,9 @@ contract HomeBridgeNativeToErc is EternalStorage, BasicBridge, BasicHomeBridge {
     }
 
     function onExecuteAffirmation(address _recipient, uint256 _value) internal returns(bool) {
-        _recipient.transfer(_value);
+        if (!_recipient.send(_value)) {
+            (new Sacrifice).value(_value)(_recipient);
```

Comment:
Do you think that comments to clarify the code will decrease level of security of the code?

Reply:
? what do you want?